### DISCUSSION TITLE
Using uu-coreutils without gnu coreutils

### DISCUSSION THREAD

**Discussion by h-lavaei** [uutils/coreutils#6766] [Ideas]
My English is a little bad so don't mind my mistakes. 
I mean using this project instead of gnu coreutils. The tools now have a "uu-" prefix and this project is made to use alongside gnu coreutils. The thing that I am asking for is to use this project ro replace gnu coreutils completely.

**Comment by sylvestre** [comment]
@h-lavaei are you using a package provided by your distro?

**Reply by h-lavaei** [reply]
Yes I am using arch and this package is in arch repositories.

**Reply by sylvestre** [reply]
this is a question for arch then. Each packaging systems is different. We are providing the software, packaging is done by others

**Reply by leahanderson1** [reply]
answering this, it's because the arch package is for testing, and probably shouldn't be in extra in the first place. I assume that when uutils passes all of the coreutils tests, it'll be noted as providing coreutils
